docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_ToggleSwitch.htm

```diff
@@ -14,6 +14,7 @@ <h1>Toggle Switch</h1>
 
 <p>The ToggleSwitch component is used as 2-state switch for on/off or true/false data input.</p>
 
+<img src="../../images/mobile_toggleSwitch.png" width="15%">
```

Comment:
The image is broken. There should be colored list in background.

Reply:
Fixed